packages/elements-core/src/components/Docs/HttpService/HttpService.spec.tsx

```diff
@@ -31,36 +32,60 @@ describe('HttpService', () => {
     expect(wrapper.getByText(httpService.name).tagName.toLowerCase()).toBe('h1');
   });
 
-  it('displays first server url', () => {
+  it('displays all server urls', () => {
     render(<ServerInfo servers={httpService.servers ?? []} />);
 
-    const serverUrl = screen.getByLabelText('production-server');
+    const serverUrl = screen.getByLabelText('Production API');
     expect(serverUrl).toHaveTextContent('https://api.stoplight.io');
 
-    const secondServer = screen.queryByText('https://api.staging.stoplight.io');
-    expect(secondServer).not.toBeInTheDocument();
+    const secondServerUrl = screen.getByLabelText('Staging API');
+    expect(secondServerUrl).toHaveTextContent('https://api.staging.stoplight.io');
 
-    expect(screen.queryByLabelText('mock-server')).not.toBeInTheDocument();
+    const thirdServerUrl = screen.getByLabelText('Integration API');
+    expect(thirdServerUrl).toHaveTextContent('https://api.int.stoplight.io');
+
+    const fourthServerUrl = screen.getByLabelText('Development API');
+    expect(fourthServerUrl).toHaveTextContent('https://localhost:4060');
+
+    expect(screen.queryByLabelText('Mock Server')).not.toBeInTheDocument();
+  });
+
+  it('generates names for servers without names', () => {
+    render(<ServerInfo servers={httpServiceWithUnnamedServers.servers ?? []} />);
+
+    const serverUrl = screen.getByLabelText('Production API');
+    expect(serverUrl).toHaveTextContent('https://api.stoplight.io');
```

Comment:
I'd not assert these here again. It's already tested above, and in general one test should have one reason to fail. Just test what the test title says.